OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -932,7 +922,10 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 			else if (order.OrderString == "Stop")
 				self.CancelActivity();
 			else if (order.OrderString == "Scatter")
-				Nudge(self);
+			{
+				self.QueueActivity(order.Queued, new Nudge(self, self));
+				self.ShowTargetLines();
```

Comment:
I'm not 100% convinced that this is the correct thing to do for `TargetLinesType.Manual` - this usually only displays lines to the specifically targeted location (i.e. under the mouse cursor), and not for automatic actions where the actor decides itself where to go.

What do others think?

Reply:
This is how every other move works

Reply:
`Manual` is supposed to only show you target lines when you ask it to. What does it have to do with targets or the action type?

Reply:
This part fixes #17036
See the handling of move order above. IMO it should show target lines for all orders that make the target move

Reply:
This code path is just for manually given orders by the player, not for units that nudge automatically (e.g. moving out of the way in front of a production building).

Reply:
> This code path is just for manually given orders by the player, not for units that nudge automatically (e.g. moving out of the way in front of a production building).

With this PR target lines are displayed for automatically nudged units. 

> I'm not 100% convinced that this is the correct thing to do for `TargetLinesType.Manual` - this usually only displays lines to the specifically targeted location (i.e. under the mouse cursor), and not for automatic actions where the actor decides itself where to go.
>

@pchote that's a valid concern but we already have some precedents that violate this principle (e.g. aircraft returning to base, units automatically choosing repair pads). The reason I'm not convinced that scatter commands needs target lines is that they don't bring much value and mostly cause visual noise. But I'm strongly against adding them.

Reply:
>> This code path is just for manually given orders by the player, not for units that nudge automatically (e.g. moving out of the way in front of a production building).

> With this PR target lines are displayed for automatically nudged units.

not automatically

Reply:
Every other move order displays move commands, it's rather jarring when move commands given by scatter don't

Reply:
> IMO target lines should be shown for all orders that move the target

I agree with that - as a player I wanna know where a unit is going even if it was an automatic order.